AY2021S2-CS2103-T14-1/tp
# Implement tests for commands associated with Room

### What this does:
Increases code coverage for commands associated with the room class
Coincidentally closes #69 

### How to test:
1. Checkout the project
2. Run tests for the project
3. Ensure all tests are passing
